components/ui/ModalConfirmation.module.css

```diff
@@ -0,0 +1,13 @@
+.customOverlay {
```

Comment:
I'm not a fan of creating a new file (+ not consistent with the project), especially since this project is using a mix of styled components and tailwindcss. 

We really need to pick just one... 😅  (but thats another topic for another day)

1. Can you either use styled components or use tailwindcss in the `ModalConfirmation` file 
2. then delete this file